cms/static/js/spec/views/video_thumbnail_spec.js

```diff
@@ -149,38 +186,152 @@ define(
+                expect($thumbnailErrorEl.find('.action-text').html().trim()).toEqual(errorMessage);
+            });
+
+            it('should show error message when file is larger than maximum size', function() {
+                var errorMessage,
+                    $thumbnailErrorEl,
+                    videoListView = createVideoListView(),
+                    $videoListEl = videoListView.render().$el,
+                    videoThumbnailView = videoListView.itemViews[0].videoThumbnailView, // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MAX_BYTES + 10)]});
+
+                errorMessage = 'The selected image must be smaller than ' +
+                    videoThumbnailView.getVideoImageMaxSize().humanize + '.';
+                // Verify error message
+                $thumbnailErrorEl = $videoListEl.find('.thumbnail-error-wrapper');
+                expect($thumbnailErrorEl.find('.action-text').html().trim()).toEqual(errorMessage);
+            });
+
+            it('should not show error message when file is appropriate size', function() {
+                var videoListView = createVideoListView(),
+                    $videoListEl = videoListView.render().$el,
+                    videoThumbnailView = videoListView.itemViews[0].videoThumbnailView, // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MIN_BYTES)]});
+
+                // Verify error not present.
+                expect($videoListEl.find('.thumbnail-error-wrapper')).not.toExist();
+            });
+
+            it('should show error message when file has unsupported content type', function() {
+                var errorMessage,
+                    $thumbnailErrorEl,
+                    videoListView = createVideoListView(),
+                    $videoListEl = videoListView.render().$el,
+                    videoThumbnailView = videoListView.itemViews[0].videoThumbnailView, // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
+
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MIN_BYTES, 'mov/mp4')]});
+
+
+                errorMessage = 'This image file type is not supported. Supported file types are ' +
```

Comment:
@muhammad-ammar I am verifying meesage from the view below, this is just a a variable to hold expected error message